### PR TITLE
fix(vant-cli): fix Windows system build style problems

### PR DESCRIPTION
**Pull request by yinhui1129754**
### Before submitting a pull request, please make sure the following is done:

1. Read the [contributing guide](https://github.com/youzan/vant/blob/dev/.github/CONTRIBUTING.md).
2. If you've added code that should be tested, add tests.
3. If you've changed APIs, update the documentation.
4. Ensure the test suite passes (`npm test`).

#### Title Format

type(ComponentName?)：commit message

Example：

- docs: fix typo in quickstart
- build: optimize build speed
- fix(Button): incorrect style
- feat(Button): add color prop

Allowed Types:

- fix
- feat
- docs
- perf
- test
- types
- build
- chore
- refactor
- breaking change

### CODE REVIEW

File: packages/vant-cli/src/compiler/gen-site-desktop-shared.ts [hunk 1]
```diff
@@ -65,7 +65,7 @@ function resolveDocuments(components: string[]): DocumentItem[] {
     });
   }
 
-  const staticDocs = glob.sync(join(DOCS_DIR, '**/*.md')).map(path => {
+  const staticDocs = glob.sync(normalizePath(join(DOCS_DIR, '**/*.md'))).map(path => {
```
Comment (chenjiahan) [inline packages/vant-cli/src/compiler/gen-site-desktop-shared.ts]: 不应该包含之前 PR 的改动

File: packages/vant-cli/src/compiler/gen-style-deps-map.ts [hunk 2]
```diff
@@ -1,15 +1,14 @@
-import { join } from 'path';
+import nodePath, { join } from 'path';
```
Comment (chenjiahan) [inline packages/vant-cli/src/compiler/gen-style-deps-map.ts]: 可以直接写成`import { sep, join, relative } from 'path';`

下同

File: packages/vant-cli/src/compiler/gen-component-style.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@
  * Build style entry of all components
  */
 
-import { join, relative } from 'path';
+import pathNode, { join, relative } from 'path';
```
Comment (chenjiahan) [inline packages/vant-cli/src/compiler/gen-component-style.ts]: 可以直接写成`import { sep, join, relative } from 'path';`

File: packages/vant-cli/src/common/index.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { join } from 'path';
+import pathNode, { join } from 'path';
```
Comment (chenjiahan) [inline packages/vant-cli/src/common/index.ts]: 可以直接写成`import { sep, join } from 'path';`

File: packages/vant-cli/src/compiler/gen-site-desktop-shared.ts (hunk 1 above)
Comment (yinhui1129754) [reply packages/vant-cli/src/compiler/gen-site-desktop-shared.ts]: 如何不让之前的分支给提交上来呀？？？

Comment (chenjiahan) [reply packages/vant-cli/src/compiler/gen-site-desktop-shared.ts]: 参见 https://youzan.github.io/vant/#/zh-CN/contribution#pull-request-liu-cheng

可以同步下最新代码

**Pull request closed without merging**